loginpage/js/loginpage.js

```diff
@@ -28,7 +28,7 @@ $(document).ready(function()
     $("#remember").click(function(){setRemember();});
```

Comment:
Opening brace must be the last content on the line

*PHPCodeSnifferBear, severity NORMAL, section `js`.*